Apps/Admin/Server/Controllers/SupportController.cs

```diff
@@ -116,6 +116,7 @@ public async Task<IEnumerable<PatientSupportResult>> GetPatients([FromQuery] Pat
                     includeEverything,
                     includeEverything,
                     includeEverything,
+                    includeEverything,
```

Comment:
https://stackoverflow.com/questions/25302178/using-async-tasks-with-the-builder-pattern

Reply:
Alternatively, we could create a model and pass the model, e.g.

```
{
    QueryType: queryType,
    QueryString: queryString,
    IncludeMessagingVerification: userIsAdmin || userisReviewer,
    IncludeBlockedDataSources: userIsAdmin || userisReviewer,
    IncludeAgentActions: userIsAdmin || userisReviewer,
    IncludeDependents: userIsAdmin || userisReviewer,
    IncludeCovidDetails: userIsSupport,
    IncludeDependents: userIsAdmin || userisReviewer,
    RefreshVaccineDetails: refreshVaccineDetails,
}
```

Either way, I think it's worth creating a separate task since it will involve a number of changes for the unit tests.

Reply:
I'll create a task for that